### PR TITLE
[JA] Update First-Party Sets translation

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
Updates Japanese translation of privacy-sandbox/First-Party Sets 
CC: @agektmr

### CODE REVIEW

File: site/ja/docs/privacy-sandbox/first-party-sets/index.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
 ---
-layout: 'layouts/doc-post.njk'
-title: 'First-Party Sets'
-subhead: >
+layout: layouts/doc-post.njk
+title: First-Party Sets
+subhead: |2
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/first-party-sets/index.md:4]: Can you fix these kind of glicthes?

File: site/ja/docs/privacy-sandbox/first-party-sets/index.md [hunk 2]
```diff
@@ -67,19 +69,17 @@ First-Party Sets を補完する提案は、`SameParty` Cookie 属性です。 C
...
 ## 詳細
 
-* [First-Party Sets 属性と sameParty 属性](/blog/first-party-sets-sameparty/)
-* [First-Party Sets の技術 Explainer](https://github.com/privacycg/first-party-sets)
-* [Chrome プラットフォームのステータス](https://chromestatus.com/feature/5640066519007232)
-* [Chromium プロジェクト](https://www.chromium.org/updates/first-party-sets)
+- [First-Party Sets 属性と sameParty 属性](/blog/first-party-sets-sameparty/)
+- [First-Party Sets の技術 Explainer](https://github.com/privacycg/first-party-sets)
+- [Chrome プラットフォームのステータス](https://chromestatus.com/feature/5640066519007232)
+- [Chromium プロジェクト](https://www.chromium.org/updates/first-party-sets)
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/first-party-sets/index.md:85]: I didn't change any of lines above which turns `*` into `-`. LIne 76-78 and 82-85. Are these done by GitLocalize?

**Review by agektmr — CHANGES_REQUESTED** [review]

File: site/ja/docs/privacy-sandbox/first-party-sets/index.md (hunk 1 above)
Comment (ilyaspiridonov) [reply site/ja/docs/privacy-sandbox/first-party-sets/index.md]: done!

File: site/ja/docs/privacy-sandbox/first-party-sets/index.md (hunk 2 above)
Comment (ilyaspiridonov) [reply site/ja/docs/privacy-sandbox/first-party-sets/index.md:81]: Correct, by GitLocalize. Both `*` and `-` are the correct notations for markdown bullet lists, and GL's reverse parser uses the `-` notation. 
We figured this shouldn't be a problem as long as 11ty renders correct html from this (which it does): 
Screenshot 2022-08-26 at 15 59 11

If there's another reason this is important, please let me know and we'll look into this with a dev team (or set up a GitHub action on our fork to correct this automatically).

**Review by mihajlija — APPROVED** [review]

File: site/ja/docs/privacy-sandbox/first-party-sets/index.md (hunk 2 above)
Comment (agektmr) [reply site/ja/docs/privacy-sandbox/first-party-sets/index.md:81]: I agree this shouldn't be a problem, but I'm curious why this conversion occurred if GitLocalize only translates updated fragments. Also, I didn't touch the section at all. When did this conversion occur?

**Review by agektmr — APPROVED** [review]
LGTM

**Comment by agektmr** [comment]
Holding off on merging this until confirming my last comment is delivered to @ilyaspiridonov

File: site/ja/docs/privacy-sandbox/first-party-sets/index.md (hunk 2 above)
Comment (agektmr) [reply site/ja/docs/privacy-sandbox/first-party-sets/index.md:81]: I see. Let's revisit when this causes any problems.

**Comment by agektmr** [comment]
@ilyaspiridonov looks like it's complaining about CLA. Do you have any solutions for it?

**Comment by ilyaspiridonov** [comment]
@agektmr I have rebased against upstream, which removed commits authored by our fork sync bot. So this PR is now green. 
We have updated the GitHub action on our fork, so all future commits that sync our fork with upstream should be authored by a user that has a signed CLA. 
Thank you!

**Pull request merged**